rep-0105.rst

```diff
@@ -81,9 +84,31 @@ The coordinate frame called ``base_link`` is rigidly attached to the
 mobile robot base. The ``base_link`` can be attached to the base in
 any arbitrary position or orientation; for every hardware platform
 there will be a different place on the base that provides an obvious
-point of reference. Note that REP 103 [1] specifies a preferred
+point of reference. Note that REP 103 [1]_ specifies a preferred
 orientation for frames.
 
+earth
+'''''
```

Comment:
It would be great to have that problem. However I'd suggest deferring this to a later time. If people are setting up a system on another planet or celestial body it would make sense to have a slightly more generic name. And in the current units of meters, tf frames are not really appropriate at interplanetary distances. Part of this is that for standardization we're picking ECEF which is very specific on earth, and less well defined anywhere else.